frontend/src/lib/bare-metal-assets.ts

```diff
@@ -90,7 +63,7 @@ export async function syncBMAs(hosts: JsonArray, resources: JsonArray) {
             const namespace = get(secret, 'value.metadata.namespace')
             return `${name}-${namespace}`
         })
-        const setSecrets = (fhosts) => {
+        const setSecrets = (fhosts: any[]) => {
```

Comment:
Setting to any is pretty much like not setting a type at all; you should type the structure of `fhosts`

Reply:
fixed